arcgis-ios-sdk-samples/Maps/Browse building floors/BrowseBuildingFloorsViewController.swift

```diff
@@ -0,0 +1,115 @@
+// Copyright 2021 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class BrowseBuildingFloorsViewController: UIViewController {
+    // MARK: Storyboard views
+    
+    /// The map view managed by the view controller.
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            mapView.map = makeMap()
+        }
+    }
+    /// The floor level picker view.
+    @IBOutlet var floorLevelPickerView: UIPickerView!
+    
+    // MARK: Properties
+    
+    /// The levels of the floor-aware web map, with an addition of `nil`.
+    var levels: [AGSFloorLevel?] = [nil]
```

Comment:
This is the first time I have seen an array of optionals in a long time because the use case for that sort of thing is fairly rare. I can't help but wonder if there's a different solution, which I'll touch on in subsequent comments.